repository/a.json

```
@@ -452,6 +452,17 @@
 				}
 			]
 		},
+		{
+			"name": "Aethra",
```

[Color schemes](https://packagecontrol.io/browse/labels/color%20scheme) usually have "Color Scheme" in their name. It's somewhat of a convention, although it's not adhered to perfectly.

```suggestion
			"name": "Aethra Color Scheme",
```